peachjam/models/timeline.py

```diff
@@ -3,26 +3,66 @@
 from itertools import islice
 
 from django.db import models
+from django.db.models import Prefetch
 from django.db.models.functions import TruncDate
 from django.utils import timezone
 from django.utils.translation import gettext_lazy as _
 
+from peachjam.models import CoreDocument
+
 log = logging.getLogger(__name__)
 
 
+class TimelineEventManager(models.Manager):
```

Comment:
@longhotsummer i went back and forth with the best way to attach subject documents to event instances and ended up with this pattern. Its trickier than it seemed because we need to jump across multiple tables. This is the simplest and most efficent, and reusable way to get this done, that i could find